client/app/components/packages/scope-of-work-draft/show.hbs

```diff
@@ -1,71 +1,139 @@
-            </small>
-          </li>
-        {{/each}}
-      </ul>
-    </section>
-  </div>
+                {{#if (eq ceqrInvoiceQuestionnaire.dcpProjectspolelyconsistactionsnotmeasurable (optionset 'ceqrInvoiceQuestionnaire' 'dcpProjectspolelyconsistactionsnotmeasurable' 'code' 'NO'))}}
+                  <Ui::Answer @beside={{true}} as |A|>
+                    <A.Prompt>
+                      What is the Square Footage of the Total Project?
+                    </A.Prompt>
+                    <A.Field>
+                    {{optionset 'ceqrInvoiceQuestionnaire' 'dcpSquarefeet' 'label' ceqrInvoiceQuestionnaire.dcpSquarefeet}}
+                    </A.Field>
+                  </Ui::Answer>
 
-  <div class="cell large-4 sticky-sidebar">
-    <Messages::Assistance class="large-margin-top" />
-  </div>
-</div>
+                  <Ui::Answer @beside={{true}} as |A|>
+                    <A.Prompt>
+                      Does the project consist solely of modification actions that are not subject to 197c? 
+                    </A.Prompt>
+                    <A.Field>
+                      {{optionset 'ceqrInvoiceQuestionnaire' 'dcpProjectmodificationtoapreviousapproval' 'label' ceqrInvoiceQuestionnaire.dcpProjectmodificationtoapreviousapproval}}
+                    </A.Field>
+                  </Ui::Answer>
 
+                  <Ui::Answer @beside={{true}} as |A|>
+                    <A.Prompt>
+                      Does your project produce a need for a CEQR Restrictive Declaration? 
+                    </A.Prompt>
+                    <A.Field>
+                    {{optionset 'ceqrInvoiceQuestionnaire' 'dcpRespectivedecrequired' 'label' ceqrInvoiceQuestionnaire.dcpRespectivedecrequired}}
+                    </A.Field>
+                  </Ui::Answer>
+                {{/if}}
+              {{/if}}
+             </saveableForm.SaveableForm>
+            </saveableForm.Section>
+          {{/let}}
+        {{/if}}
+        
+      <saveableForm.Section
+        @title="Attached Documents" 
+        @elementId="attachments"
+        data-test-attached-documents
+      >
+        <ul class="no-bullet">
+          {{#each @package.documents as |document idx|}}
+            <li class="ruled-adjacent tight">
+              <a
+                href={{concat (get-env-variable 'host') '/documents' document.serverRelativeUrl}}
+                target="_blank"
+                rel="noopener noreferrer"
+                data-test-document-name={{idx}}
+              >
+                <strong>
+                  {{document.name}}
+                </strong>
+              </a>
+              <small class="text-gray">
+                {{document.timeCreated}}
+              </small>
+            </li>
+          {{/each}}
+        </ul>
+      </saveableForm.Section>
+    </div>
+
+    <div class="cell large-4 sticky-sidebar">
+      <Messages::Assistance class="large-margin-top" />
+      <saveableForm.PageNav>
```

Comment:
Since there is no content inside this component, I think you can just write
`<saveableForm.PageNav />` 

instead of the opening and closing tags 
```
  <saveableForm.PageNav>
  </saveableForm.PageNav>
```

Reply:
Fixed